src/v2/Apps/ViewingRoom/ViewingRoomApp.tsx

```diff
@@ -48,16 +48,18 @@ const ViewingRoomApp: React.FC<ViewingRoomAppProps> = ({
   const getView = () => {
     if (!user) {
       return
```

Comment:
maybe this should return the `ViewingRoomContentNotAccessible` component? We could have the `ViewingRoomContentNotAccessible` handle text for logged out state. And with this refactor work, it might be more readable to move this logic back into the `render` function and get rid of this `getView()` helper function.